### PR TITLE
Use debounce for dispatch scrollTopNotification and expandNotifications

### PR DESCRIPTION
**Pull request by unarist**
|Before|After|
|-|-|
|gif|gif2|

### CODE REVIEW

**Comment by Gargron** [comment]
Perhaps this is a case for debounce? I don't like storing state outside of React (it is against the React philosophy, only acceptable for corner cases)

**Comment by unarist** [comment]
Ah, I found similar fixes using debounce in our code. I will rewrite with that.

**Review by Gargron — APPROVED** [review]

File: app/javascript/mastodon/features/notifications/index.js [hunk 1]
```diff
@@ -72,7 +77,7 @@ class Notifications extends React.PureComponent {
 
   handleLoadMore = (e) => {
     e.preventDefault();
-    this.props.dispatch(expandNotifications());
+    dispatchExpandNotifications();
```
Comment (Gargron) [inline app/javascript/mastodon/features/notifications/index.js]: Should be `this.dispatchExpandNotifications()`

**Review by Gargron — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/features/notifications/index.js (hunk 1 above)
Comment (unarist) [inline app/javascript/mastodon/features/notifications/index.js]: Oops, I've rebased and fixed it.

**Review by Gargron — APPROVED** [review]

**Pull request merged**